quarkus-workshop-super-heroes/super-heroes/ui-super-heroes/src/main/webui/src/styles.css

```diff
@@ -121,14 +119,15 @@ td {
   10px 0 50px #FF5B68;
 }
 
-.row {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-}
-
 #fight-row {
-  margin-bottom: 30px;
+    display: grid;
+    grid-template-columns: 2fr 1fr 2fr;
```

Comment:
I think `grid-template-columns: 35% 30% 35%;` might be better?

Reply:
I'm not an expert, at all, but my understanding (from hastily reading https://stackoverflow.com/questions/45090726/the-difference-between-percentage-and-fr-units) is that fr is a proportion of the free space and percentages are a percentage of the absolute space, so if you have wide gaps and use a percentage it could overflow. 
But if the aim is just to get them more equal, something like 1.2fr 1fr 1.2fr might be good.

Reply:
You're more of an expert than I am! I think your suggestion looks good.